### DISCUSSION TITLE
Metronome: Visual Feedback

### DISCUSSION THREAD

**Discussion by Gianclgar** [InfiniTimeOrg/InfiniTime#2115] [Ideas]
Add visual feedback to the metronome (eg. screen blinks, background changes color to white, etc) so users won't rely only on haptic feedback. Could also include weakest changes for secondary beats, following the vibration

**Comment by L3br4nd** [comment]
I m a drummer and I prefer visual metronomes to check the tempo occasionally during playing. I would like the visual option, too as an alternative and/or addition to the haptic feedback.
